pkg/agent/config.go

```diff
@@ -55,12 +55,13 @@ type Config struct {
 	KafkaBrokers []string `env:"KAFKA_BROKERS" envSeparator:","`
 	// KafkaTopic is the name of the topic where the flows' processor will receive the flows from.
 	KafkaTopic string `env:"KAFKA_TOPIC" envDefault:"network-flows"`
-	// KafkaBatchSize sets the limit on how many messages will be buffered before being sent to a
+	// KafkaBatchMessages sets the limit on how many messages will be buffered before being sent to a
 	// partition.
-	KafkaBatchSize int `env:"KAFKA_BATCH_SIZE" envDefault:"100"`
-	// KafkaBatchBytes sets the limit of the maximum size of a request in bytes before being sent
+	KafkaBatchMessages int `env:"KAFKA_BATCH_MESSAGES" envDefault:"1000"`
```

Comment:
do we get better results with `1000` here ?

Reply:
Yes, a small, idle cluster is generating about 500 flows on each period. That would mean 5 batches to be sent to Kafka. For big clusters, we observed around 10.000 flows on each batch (around 100 batches to Kafka). Increasing that to 1000 would mean less interactions with kafka (1 and 10 batches, respectively)

Reply:
makes sense :+1: